README.md

```diff
@@ -32,6 +32,7 @@ We want examples of as many use cases in this repository as possible! Submit a P
 * [Sumo Logic](examples/fluent-bit/sumologic)
 * [SolarWinds Loggly](examples/fluent-bit/solarwinds-loggly)
 * [Sematext Logs](examples/fluent-bit/sematext)
+* [Send to Logstash](examples/fluent-bit/logstash)
```

Comment:
nit: make it just "Logstash". The others don't have a "Send to", they just have the destination name